hub/policy/src/main/java/uk/gov/ida/hub/policy/PolicyApplication.java

```diff
@@ -66,11 +67,19 @@ public final void initialize(Bootstrap<PolicyConfiguration> bootstrap) {
         bootstrap.addBundle(new LoggingBundle());
         bootstrap.addBundle(new PrometheusBundle());
         bootstrap.addBundle(new IdaJsonProcessingExceptionMapperBundle());
+
         final InfinispanBundle infinispanBundle = new InfinispanBundle();
-        // the infinispan cache manager needs to be lazy loaded because it is not initialized at this point.
         bootstrap.addBundle(infinispanBundle);
+
+        final SessionBundle sessionBundle = new SessionBundle(infinispanBundle.getInfinispanCacheManagerProvider());
+
+        // the infinispan cache manager needs to be lazy loaded because it is not initialized at this point.
+        bootstrap.addBundle(sessionBundle);
         guiceBundle = GuiceBundle.defaultBuilder(PolicyConfiguration.class)
-                .modules(getPolicyModule(), new EventEmitterModule(),  bindInfinispan(infinispanBundle.getInfinispanCacheManagerProvider()))
+                .modules(getPolicyModule(),
+                        new EventEmitterModule(),
+                        getSessionModule(infinispanBundle.getInfinispanCacheManagerProvider(),
```

Comment:
Can we replace this line with `new SessionModule(infinispanBundle.getInfinispanCacheManagerProvider(), sessionBundle.getSessionStateStoreProvider())` and remove `getSessionModule()` method from this class? Less code -> easy to read.